certificate-service/app/src/main/java/se/inera/intyg/certificateservice/infrastructure/certificatemodel/fk3226/CertificateModelFactoryFK3226.java

```diff
@@ -150,8 +161,137 @@ public class CertificateModelFactoryFK3226 implements CertificateModelFactory {
   public static final SchematronPath SCHEMATRON_PATH = new SchematronPath(
       "fk3226/schematron/lunsp.v1.sch");
 
-  public static final String PDF_FK_3226_PDF = "fk3226/pdf/fk3226_v1.pdf";
-  public static final String PDF_NO_ADDRESS_FK_7472_PDF = "fk3226/pdf/fk3226_v1_no_address.pdf";
+  public static final String PDF_FK_3226_PDF = "fk3226/pdf/fk3226_v1_no_address.pdf"; //TODO: update when new template has been provided
+  public static final String PDF_NO_ADDRESS_FK_3226_PDF = "fk3226/pdf/fk3226_v1_no_address.pdf";
+  public static final Mcid PDF_MCID = new Mcid(100);
+  private static final int PDF_SIGNATURE_PAGE_INDEX = 1;
```

Comment:
Maybe it's better to move all PDF-constants to a separate class since there are so many. But since we discussed that all certificate type specific code should be placed in or as close to the model as possible i decided to let them stay here for now.

Reply:
I think you are on to something. Our factory-class has grown quite big and it could do with some refactoring, but it isn't necessary to do it in this PR.